lib/math.c

```diff
@@ -0,0 +1,9 @@
+/* libc: Math function. */
+
+#include <lib/libc.h>
+
+/* Returns the absolute value of an integer. */
+int32_t abs(int32_t n)
+{
+    return n >= 0 ? n : -n;
+}
```

Comment:
Be aware of newline.
See https://medium.com/@alexey.inkin/how-to-force-newline-at-end-of-files-and-why-you-should-do-it-fdf76d1d090e